pippo-core/src/main/java/ro/pippo/core/Pippo.java

```diff
@@ -245,7 +245,10 @@ public void onEvent(ReloadWatcher.Event event, Path dir, Path path) {
     }
 
     private void addShutdownHook() {
-        Runtime.getRuntime().addShutdownHook(new Thread(Pippo.this::stop));
```

Comment:
I agree about stop. My thinking was that stop logs a warning when you call it twice, so I should be sure not to call it twice. But maybe the better change is to change the log to `debug` since calling stop twice is normal?